### PR TITLE
(TFECO-8260) Add Heimdall metadata

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

File: META.d/_summary.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+schema: 1.1
+partition: tf-ecosystem
+category: library
```
Comment (austinvalle) [inline META.d/_summary.yml:6]:
```suggestion
category: github-action
```

Looks like GHA have their own category: https://heimdall.hashicorp.services/site/assets?category=github-action

File: META.d/_summary.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+summary:
+  owner: team-tf-editor-experience
```
Comment (austinvalle) [inline META.d/_summary.yml:9]:
```suggestion
  owner: team-tf-devex
```

We don't have a shared team ATM, but this matches the CODEOWNERS: https://heimdall.hashicorp.services/site/teams/team-tf-devex

**Review by austinvalle — COMMENTED** [review]
Thanks for submitting this @jpogran! I have two suggestions in the comments

File: META.d/_summary.yml (hunk 2 above)
Comment (jpogran) [reply META.d/_summary.yml:9]: I was following the ticket, but will definitely cede to the registered owner!

**Review by austinvalle — APPROVED** [review]
Thank you! 🚀

**Pull request merged**